setup.py

```diff
@@ -0,0 +1,8 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="pyArborX",
```

Comment:
I did not really find anything like guidelines. Most of the names seem to be lowercase. We can call it arborx if we want, sure, that is an easy change. The repository (and thus the main folder) are already called `pyArborX` which might be confusing for some users but I don't know how bad this is in reality

Reply:
let me know if you want me to change it

Reply:
I personally would prefer `arborx`, so that one could write:
```python
import scikitlearn
import arborx
```
or
```shell
pip install arborx
```
But I'm not going to block.

Reply:
I will change it

Reply:
done

Reply:
I actually did not mean for you to change anything other than the name of the export Python package name. I think internally we could/should still use `pyArborX`.

Reply:
I tried just renaming it in the `setup.py` but it does not work. the module is only found if it its files match the name in the `setup.py`

Reply:
Would it be fine to use the name `pyArborX` and just tell users to import it with `import pyArborX as arborx`? This way we could have the files in the repo/namespace/filename_prefix/Cmake_vars be similar but in examples still do calls via `arborx.BVH`